bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZ_leptonic/WLLjj_dim8_ewk_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model QCKM_5_Aug21v2
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+generate    p p > w+ z j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w+ > l+ vl, z > l+ l- @ 1
+add process p p > w- z j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w- > l- vl~, z > l+ l- @ 2
```

Comment:
do you want l+ include ta+, if yes, you may need to define like ell+ = l+ ta+, and use z > ell+ ell-

Reply:
Hi @menglu21 yes, I have added definitions of taus. Thanks